extensions/wikibase/module/scripts/schema-alignment.js

```diff
@@ -433,8 +453,19 @@ SchemaAlignment._changesCleared = function() {
         .addClass('disabled');
 };
 
-SchemaAlignment._createDraggableColumn = function(name, reconciledSiteIRI) {
+SchemaAlignment._createDraggableColumn = function(name, reconciledSiteIRI, reconConfig) {
+  var logo=serviceLogoFromReconConfig(reconConfig);
   var cell = $("<div></div>").addClass('wbs-draggable-column').text(name);
+  if(logo) {
+    var img =$("<img>");
+    if(serviceLogo) {
```

Comment:
This `if` block is probably redundant with the outer one, and seems to refer to an undeclared variable.